OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -338,15 +338,24 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 
 				default:
 					{
-						if (order.Subject != null && !order.Subject.IsDead)
-							foreach (var t in order.Subject.TraitsImplementing<IResolveOrder>())
-								t.ResolveOrder(order.Subject, order);
+						if (!order.Grouped)
+							ResolveOrder(order);
+						else
+							foreach (var subject in order.ExtraActors)
```

Comment:
We should implement a new `order.GroupActors` instead of overloading `ExtraActors` for this - there may be orders that want to use both.